packages/react-reconciler/src/ReactFiberCompleteWork.new.js

```diff
@@ -1062,6 +1062,13 @@ function completeWork(
                 // Reset the effect list before doing the second pass since that's now invalid.
                 if (renderState.lastEffect === null) {
                   workInProgress.firstEffect = null;
+                  workInProgress.subtreeTag = NoEffect;
+                  // TODO (effects) This probably isn't the best approach. Discuss with Brian
```

Comment:
@lunaruan You left a TODO to talk to me here. Still relevant?

Reply:
Nope we're good :D